jota/src/main/java/jota/IotaAPI.java

```diff
@@ -732,13 +719,17 @@ public GetAccountDataResponse getAccountData(String seed, int security, int inde
             throws InvalidBundleException, ArgumentException, InvalidSignatureException,
             InvalidTrytesException, InvalidSecurityLevelException, InvalidAddressException, NoInclusionStatesException, NoNodeInfoException {
 
+        if (start > end || end > (start + 1000)) {
```

Comment:
Could we just provide an overload with default end parameter and remove the upper bound check here?

Reply:
Good question, is the upper bound check necessary?